packages/components/src/modal/index.js

```diff
@@ -36,8 +36,8 @@ export default function Modal( {
 	focusOnMount = true,
 	shouldCloseOnEsc = true,
 	shouldCloseOnClickOutside = true,
-	isDismissable = true, // Deprecated
-	isDismissible = isDismissable,
+	isDismissable, // Deprecated
+	isDismissible = isDismissable || true,
```

Comment:
Not sure I get this. Does that mean `isDismissable` can't be `false` at all? If so, what's the point of having this expression? Also, I see `isDismissable` is used to toggle the `Cancel` button. I think that we should be able to hide this button, as some modals will have their own button configurations.